features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Endpoints/GeneralConfiguration/Certificates.jsx

```diff
@@ -195,11 +202,16 @@ function Certificates(props) {
                                             <Icon>info</Icon>
                                         </IconButton>
                                         <IconButton
+                                            disabled={isRestricted(['apim:api_create'], apiFromContext)}
                                             onClick={() => setCertificateToDelete({ open: true, alias: cert.alias })}
                                         >
-                                            <Icon color='error'>
-                                                delete
-                                            </Icon>
+                                            {isRestricted(['apim:api_create'], apiFromContext) ? (
+                                                <Icon className={classes.deleteIconDisable} > delete
```

Comment:
can't we have one Icon with className set by a ternery operator?
`<Icon className={isRestricted(['apim:api_create'], apiFromContext) ? classes. deleteIconDisable : classes.deleteIcon} >  delete  </Icon>`

Reply:
fixed